ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/perception/RDXStancePoseSelectionPanel.java

```diff
@@ -106,8 +115,8 @@ private void updatePoses()
             {
                for (int i = 0; i < leftPoses.size(); i++)
                {
-                  leftSpheres.add(RDXModelBuilder.createSphere(0.015f, Color.BLUE));
-                  rightSpheres.add(RDXModelBuilder.createSphere(0.015f, Color.RED));
+                  leftSpheres.add(RDXModelBuilder.createSphere(0.01f, Color.WHITE));
+                  rightSpheres.add(RDXModelBuilder.createSphere(0.01f, Color.BLACK));
```

Comment:
should colors match the red/green we use for left/right?

Reply:
They were red and green but I found that when the height map is one of those colors the grid's are really difficult to see. I don't have a preference on the color just wanted to make it obvious to see